tests/FVTTests/test/testFunctions.js

```diff
@@ -0,0 +1,366 @@
+    console.log('got file links');
+
+    // Find the jobFileName we're looking for from the list of job files
+    for (const fileLink of fileLinks) {
+        const text = await fileLink.getText();
+        console.log(`file link text: ${text}`);
+        if (text === jobFileName) {
+            await fileLink.click();
+            console.log('clicked file link');
+            break;
+        }
+    }
+    // Check the job name/prefix is in the content
+    for (let i = 0; i < 15; i++) {
+        const textviewContent = await driver.findElements(By.className('textviewContent'));
+        console.log('got textviewContent');
+        const text = await textviewContent[0].getText(textviewContent);
+        console.log(`text view content: ${text}`);
+        if (!text.includes(prefixFilter)) {
+            await driver.sleep(1000);
+            console.log('text didnt include prefixFilter, sleeping');
+        } else {
+            console.log('text inncluded prexif filter');
+            break;
+        }
+    }
+    return true;
+}
+
+const testHighlightColorByClass = (colorClass, elems) => {
+    const colorVal = textHighlightColors[colorClass];
+
+    let classStr = colorClass;
+    if (colorClass === NO_CLASS) {
+        classStr = '';
+    }
+
+    const colorArray = [
+        ...new Set(
+            elems
+                .filter(e => {
+                    return e.elemCss === classStr;
+                })
+                .map(e => {
+                    return e.elemColor;
+                }),
+        )];
+    let testColor = colorArray.length === 1;
+    testColor = testColor && colorArray[0] === colorVal;
+    return testColor;
+};
+
+const testAllHighlightColor = elems => {
+    let testHighlights = true;
+    textColorClasses.forEach(colorClass => {
+        testHighlights = testHighlights && testHighlightColorByClass(colorClass, elems);
+    });
+
+    return testHighlights;
+};
+
+const testFilterDisplayStringValues = async (driver, expectedFilters) => {
+    const actualFilters = await waitForAndExtractFilters(driver);
+    return compareFilters(actualFilters, expectedFilters);
+};
+
+const testFilterFormInputValues = async (driver, expectedFilters) => {
+    const element = await driver.findElement(By.id('filter-view'));
+    await element.click();
+    const actualFilters = await getAllFilterValues(driver);
+    return compareFilters(actualFilters, expectedFilters);
+};
+
+
+const testJobUrlFilters = checkFunc => {
+    return async (driver, expectedVals) => {
+        const jobs = await waitForAndExtractJobs(driver);
+        const isExpected = await checkFunc(jobs, expectedVals);
+        return isExpected;
+    };
+};
+
+const testJobOwnerFilter = testJobUrlFilters(checkJobsOwner);
+const testJobPrefixFilter = testJobUrlFilters(checkJobsPrefix);
+const testJobStatusFilter = testJobUrlFilters(checkJobsStatus);
+const testJobIdFilter = testJobUrlFilters(checkJobsId);
+
+
+module.exports = {
```

Comment:
may be we can migrate to newer javascript ES6 rather than ES5
then we can simply put export in public functions, rather than huge list at bottom

Reply:
I've experimented with this but was unable to find the correct configuration and have therefore raised https://github.com/zowe/zlux/issues/372 to track this effort